data-plane/config/brokerv2/500-dispatcher.yaml

```diff
@@ -155,6 +159,9 @@ spec:
         - name: config-kafka-broker-data-plane
           configMap:
             name: config-kafka-broker-data-plane
+        - name: kafka-broker-brokers-triggers
+          configMap:
+            name: kafka-broker-brokers-triggers
```

Comment:
This volume shouldn't be needed here.

Reply:
Same as https://github.com/knative-sandbox/eventing-kafka-broker/pull/2613#discussion_r984801145 @pierDipi

Reply:
I see, so we need to migrate them to use the generic values as the source is using:
https://github.com/knative-sandbox/eventing-kafka-broker/blob/d0d8aeb62f813ba69018656d2ab92475ac864cb8/data-plane/config/source/500-dispatcher.yaml#L95-L96

Reply:
but we should still remove the volume here, see https://github.com/knative-sandbox/eventing-kafka-broker/blob/d0d8aeb62f813ba69018656d2ab92475ac864cb8/data-plane/config/source/500-dispatcher.yaml#L155-L166

Reply:
see also how it works here https://github.com/knative-sandbox/eventing-kafka-broker/blob/main/control-plane/pkg/apis/core/pod_defaulting.go, in particular these lines:
https://github.com/knative-sandbox/eventing-kafka-broker/blob/d0d8aeb62f813ba69018656d2ab92475ac864cb8/control-plane/pkg/apis/core/pod_defaulting.go#L85-L87